### DISCUSSION TITLE
How can I send Requests in WASM?

### DISCUSSION THREAD

**Discussion by EarzuChan** [unoplatform/uno#10789] [Q&A]
I planed to use this Uno Platform to create a cross-platfrom app, but since I need to send Http Request in my app,  and it doesn't work in the browser with wasm, how to sovle this?

**Comment by jeromelaban** [comment]
You can use HttpClient to send requests like you would in any other apps, but you'll need to make sure that the remote endpoint supports CORS. See [this walkthrough for more details](https://platform.uno/docs/articles/howto-consume-webservices.html).
